src/com/foam/demos/addClass/Controller.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license Copyright 2018 The FOAM Authors. All Rights Reserved.
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'com.foam.demos.addClass',

```

Comment:
We don't own the foam.com domain, so we shouldn't use that package. Either use com.foamdev or just foam.